webpack.config.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable */
```

Comment:
- Please, do not disable the linter, but rather find out how they can be solved. This will improve your project performance